taskcluster/kinds/beetmover-ship/kind.yml

```diff
@@ -19,7 +19,7 @@ task-defaults:
     from-deps:
         kinds: [beetmover-promote]
         group-by: all
-        set-name: false
+        set-name: null
```

Comment:
I'm curious why do we need null here instead of false?

Reply:
I think it was an oversight / bug in the refactor around this.